Example/Sources/Data Generation/SampleData.swift

```diff
@@ -189,6 +190,9 @@ final internal class SampleData {
         case .Video:
             let image = messageImages.random()!
             return MockMessage(thumbnail: image, user: user, messageId: uniqueID, date: date)
+        case .Gif:
+            let image = messageImages.random()!
+            return MockMessage(thumbnail: image, user: user, messageId: uniqueID, date: date, streamingUrl: "https://www.radiantmediaplayer.com/media/big-buck-bunny-360p.mp4")
```

Comment:
why does it have mp4 extension if it's supposed to be gif?

Reply:
I think for better performance, we can use mute video.
This method does not require a **GIF** player and any new dependency, 
But if you think it must be **GIF** format, I will do it.

Reply:
Well, then it's a qurstion. Are we adding support dor videos that will be muted, or for .gif images?

Reply:
We are supporting video formats.

Reply:
Thanks for the info. :)

But we do have video type, right? Cell media type. And you can adjust it to be silent. I don't think we should merge this PR as gif support as it's not really that...

Reply:
:))) That's right ٫ so I will add a gif player instead of a video player to support the gif format

Reply:
You're welcome,

I think it is good to use **PINRemoteImage**.

Reply:
I think based on https://github.com/MessageKit/MessageKit/issues/187#issuecomment-333253813 we shouldn't really do that... as I said, adding gif support on project side is super easy, not sure if it's worth it doing it here in this framework. It would add dependency for everywhere even tho only some users would need it...